python/tank/bootstrap/manager.py

```diff
@@ -59,6 +59,8 @@ def __init__(self, sg_user=None):
                         authentication, simply construct an explicit user object
                         and pass it in.
         :type sg_user: :class:`~sgtk.authentication.ShotgunUser`
+        :param bool allow_config_overrides: Whether to allow the config override
+            environment variable to affect the results of configuration resolutions.
```

Comment:
one more thought: almost everything else on this class is a parameter rather than passed via the constructor - i *think* this one could be a good fit for that too, e.g.:

```
bootstrapper = ToolkitManager()
bootstrapper.enable_global_config_resolve = False
```